app/helpers/validate.js

```diff
@@ -0,0 +1,42 @@
+class Validate{
+
+  constructor(){
+    const passwordHelper = require('./../helpers/password');
+    this.password = new passwordHelper();
+  }
+
+  email(email){
+    if (!this.isDefined(email)) {
+      return false;
+    }
+    var
+      re = /[a-z,0-9]/ig,
+      dotPos = email.lastIndexOf('.'),
+      atPos  = email.lastIndexOf('@'),
+      wsp    = email.lastIndexOf(' '),
```

Comment:
I don't understand the point of the extra space before the `=`. If you wanted them to all be even, you're a bit off 😏